tools/api/generate_go_protobuf.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+from subprocess import check_output
+from subprocess import check_call
+import glob
+import os
+import shutil
+import sys
+import re
+
+TARGETS = '@envoy_api//...'
+IMPORT_BASE = 'github.com/envoyproxy/go-control-plane'
+OUTPUT_BASE = 'build_go'
+REPO_BASE = 'go-control-plane'
+BRANCH = 'master'
+MIRROR_MSG = 'Mirrored from envoyproxy/envoy @ '
+USER_NAME = 'Kuat Yessenov'
```

Comment:
I think we should have some kind of robo account here, e.g. what we do in https://github.com/envoyproxy/envoy/blob/master/ci/api_mirror.sh#L13.

Reply:
Is this a real account or just random email/username. I couldn't tell, and DCO didn't like it.

Reply:
Switched to a robo account.